MonoGame.Framework/Platform/Native/TitleContainer.Native.cs

```diff
@@ -10,14 +10,14 @@ namespace Microsoft.Xna.Framework;
 
 partial class TitleContainer
 {
+
     static partial void PlatformInit()
     {
     }
 
     private static Stream PlatformOpenStream(string safeName)
     {
         var absolutePath = MGP.Platform_MakePath(Location, safeName);
```

Comment:
So do we need MGP.Platform_MakePath anymore?

Do we use it other places?  Is it better for `MG.OpenRead` to take the `Location, safeName` instead and build the path internally?